addons/apecloud-mysql/values.yaml

```diff
@@ -108,3 +108,7 @@ wesqlscale:
 ## if you install multiple releases of this chart.
 ## If specified, the cluster definition will use it as name.
 resourceNamePrefix: ""
+
+etcd:
+  apeiVersion: "2"
```

Comment:
what is this filed for

Reply:
Control the API version of the local etcd, supporting either API v2 or API v3.